chromatic/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@sl-design-system/chromatic",
+  "description": "Storybook setup specifically for chromatic",
+  "version": "0.0.0",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sl-design-system/components.git",
+    "directory": "chromatic"
+  },
+  "type": "module",
+  "scripts": {
+    "chromatic": "wireit",
+    "build-storybook": "wireit",
+    "start": "wireit"
+  },
+  "wireit": {
+    "chromatic": {
+      "command": "chromatic --build-script-name build-storybook -o storybook-static --exit-zero-on-changes"
```

Comment:
If there are no dependencies etc; then this could probably be moved into `"scripts"` directly. No need for the wireit "proxy".